model/BUILD.bazel

```diff
@@ -47,6 +47,7 @@ proto_library(
 java_lite_proto_library(
     name = "event_logger_java_proto_lite",
     deps = [":event_logger_proto"],
+    visibility = ["//visibility:public"],
```

Comment:
Any reason why we are setting visibility to public?

Reply:
This is actually the intention of #2673. We want to move away from omni model group libraries and to depending on specific targets.